### ISSUE TITLE
Missing ini in mods-available for php7.x

### ISSUE THREAD

**Issue by NdK73** [oerdnj/deb.sury.org#949]
Hello.
It seems the packages for php7 modules miss the /etc/php/7.x/mods-available/modulename.ini entry. (at least the 64-bit ones I tested: mysql, mbstring, json and some others, IIRC).
Modules for 5.6 seems OK.

**Comment by oerdnj** [comment]
Please be specific. What exactly is missing in what package(s)? And how do you install the packages and how do you test the missing files (debsums?).

**Comment by NdK73** [comment]
I added the packages.sury.org repo and intall via "apt install".
But I just tried purging and reinstalling php7.2-mbstring (one of the packages that missed the ini) and now it gets created (by post-install script, so it's correctly "missing" from the deb). I'm very confused. Half an hour ago, the packages were installed and the ini files were missing...
Surely something gone wrong during the updates (possibly the files got removed by an older package). Sorry for the noise and thanks.
